src/data-types.js

```diff
@@ -87,6 +87,18 @@ export function getTypes(options) {
 		}
 	}
 
+	var int64 = {
+		unpack: function(buffer){
+			var value = buffer.readUInt64LE(buffer.pointer);
```

Comment:
Yeah, I figured Gobble would catch it if it were illegal but it doesn't seem to care. I'll try and grab a genericMesh packet and see what happens.